neqo-transport/src/cc/cubic.rs

```diff
@@ -78,33 +102,67 @@ pub struct Cubic {
+    /// > bandwidths (or small BDPs). In these networks, CUBIC remains in the Reno-friendly region to
+    /// > achieve at least the same throughput as Reno.
     ///
-    /// <https://datatracker.ietf.org/doc/html/rfc8312#section-4.2>
+    /// <https://datatracker.ietf.org/doc/html/rfc9438#name-reno-friendly-region>
+    ///
+    /// ACTION: Rename to `w_est` to conform with spec.
     estimated_tcp_cwnd: f64,
-    /// > K is the time period that the above function takes to increase the
-    /// > current window size to W_max if there are no further congestion events
+    /// > The time period in seconds it takes to increase the congestion window size
+    /// > at the beginning of the current congestion avoidance stage to `w_max`.
+    ///
+    /// <https://datatracker.ietf.org/doc/html/rfc9438#name-variables-of-interest>
     ///
-    /// <https://datatracker.ietf.org/doc/html/rfc8312#section-4.1>
+    /// For formula definition see:
+    ///
+    /// <https://datatracker.ietf.org/doc/html/rfc9438#name-window-increase-function>
     k: f64,
-    /// > W_max is the window size just before the window is reduced in the last
-    /// > congestion event.
+    /// > Size of `cwnd` in segments just before `cwnd` was reduced in the last congestion
+    /// > event when fast convergence is disabled (same as `cwnd_prior` on a congestion event).
+    /// > However, if fast convergence is enabled, `w_max` may be further reduced based on
+    /// > the current saturation point.
+    ///
+    /// <https://datatracker.ietf.org/doc/html/rfc9438#name-variables-of-interest>
+    ///
+    /// This acts as the plateau for the cubic function where it switches from the concave to the
+    /// convex region.
+    ///
+    /// For formula definition see:
     ///
-    /// <https://datatracker.ietf.org/doc/html/rfc8312#section-4.1>
+    /// <https://datatracker.ietf.org/doc/html/rfc9438#name-fast-convergence>
     w_max: f64,
-    /// > the elapsed time from the beginning of the current congestion
-    /// > avoidance
+    /// > The time in seconds at which the current congestion avoidance stage started.
     ///
-    /// <https://datatracker.ietf.org/doc/html/rfc8312#section-4.1>
+    /// <https://datatracker.ietf.org/doc/html/rfc9438#name-variables-of-interest>
+    ///
+    /// UPDATE/ACTION: Rename to `t_epoch` to adhere to spec.
     ca_epoch_start: Option<Instant>,
     /// Number of bytes acked since the last Standard TCP congestion window increase.
+    ///
+    /// UPDATE: We are using bytes but the spec recommends using segments.
+    ///
+    /// > Implementations can use bytes to express window sizes, which would require
+    /// > factoring in the SMSS wherever necessary and replacing segments_acked (Figure 4)
+    /// > with the number of acknowledged bytes.
+    ///
+    /// <https://datatracker.ietf.org/doc/html/rfc9438#name-definitions>
+    ///
+    /// ACTIONS: Discuss if we want to switch to segments, confirm we are doing all conversions.
```

Comment:
For the protocol, limited previous discussion: https://github.com/mozilla/neqo/pull/1074#discussion_r546719993